pyincore/analyses/nonstructbuildingdamage/nonstructbuildingdamage.py

```diff
@@ -428,13 +348,13 @@ def get_spec(self):
                     'id': 'result',
                     'parent_type': 'buildings',
                     'description': 'CSV file of damage states for building non-structural damage',
-                    'type': 'ergo:nsBuildingInventoryDamageVer3'
+                    'type': 'ergo:buildingDamageVer6'
```

Comment:
Even though the structure is the same as building damage (limit state and damage states), do you think we should still have a non structural damage type? Just asking since otherwise we could get rid of this analysis altogether if it produces the same output as building damage and has the same output types. But I think there is some value in knowing that is is non structural damage because otherwise from the shape of the output there is no way to tell.

Reply:
I agree. I will switch it back to the non structural type and bump the version up `nsBuildingInventoryDamageVer4`